### ISSUE TITLE
Proposal - Donate Dapr to CNCF

### ISSUE THREAD

**Issue by tomkerkhove** [dapr/dapr#1849]
## In what area(s)?

/area runtime

## Describe the proposal
Now that Dapr is getting more mature it feels time to donate Dapr to CNCF to give it a vendor-neutral home.

**Comment by mchmarny** [comment]
@tomkerkhove, thank you for opening this issue:

* What are the drivers for your proposal? 
* What issues would this address for you as a Dapr user and/or contributor?
* What impact on the project governance do you envision CNCF membership would have on Dapr?

Thank you.

**Comment by yaron2** [comment]
Hey Tom, thanks for your recommendation.  [As we have communicated](https://cloudblogs.microsoft.com/opensource/2019/10/16/announcing-dapr-open-source-project-build-microservice-applications/), we intend to donate Dapr to a vendor neutral organization.  Having documented governance is one of our prerequisites for making a project donation to a foundation.
 
We are currently developing governance and will be sharing that proposal with the community to review and offer feedback. If you are interested in participating in the development of the project governance, please let us know and we will share an early draft when it is ready.
 
Please let us know if there is something for you which is currently blocked without published governance and before moving this to a foundation.  We want to address those issues in parallel with the longer term plan to move the project to a foundation.

**Comment by tomkerkhove** [comment]
By donating it would give Dapr a more neutral place and provide governance around it that is not Microsoft specific. For some customers, this is a concern where they are afraid that Microsoft might pull the plug at one point leave their apps in a risky state while with a foundation it feels like it is less of a risk.

This will sound very harsh, and it's not my intention @msfussell, but I'd like to avoid a Service Fabric scenario where we go all onboard and focus of Microsoft is shifted to something else leaving us with a platform that has minor investment/future. That said, the fact that it's open-source already helps a lot so that if it happens; the community can fork it and proceed.

So in short; it's a concern as a user, not a contributor.

Don't get me wrong - The project is run smoothly and open, but it feels a bit Microsoft even though the platform itself is vendor-neutral.

@yaron2 Happy to help where I can so feel free to share it with me and I'll have a look!

**Comment by msfussell** [comment]
Thanks tagging this @tomkerkhove.

**Comment by tomkerkhove** [comment]
Is there an update on the progress @msfussell ?

**Comment by tomkerkhove** [comment]
What is the status of the incubation proposal @msfussell ?

**Comment by yaron2** [comment]
> What is the status of the incubation proposal @msfussell ?

Currently under CNCF due diligence

**Comment by kmaheswar** [comment]
> > What is the status of the incubation proposal @msfussell ?
> 
> Currently under CNCF due diligence

Happy to see that its now in CNCF Incubator. https://www.cncf.io/blog/2021/11/03/dapr-distributed-application-runtime-joins-cncf-incubator/
